docs/clusterManagers.html

```diff
@@ -708,12 +708,16 @@ <h1 dir="ltr">
       </li>
      </ul>
      <h3 dir="ltr">
-      Technote: TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys
+      TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys
```

Comment:
Automation setup on two Data Centers...
We are now starting on pacemaker and moving away from TSA
There are two links here.